src/core/tsi/ssl_transport_security.h

```diff
@@ -77,9 +92,9 @@ typedef struct {
      where a parameter is invalid.  */
 tsi_result tsi_create_ssl_client_handshaker_factory(
     const tsi_ssl_pem_key_cert_pair* pem_key_cert_pair,
-    const char* pem_root_certs, const char* cipher_suites,
-    const char** alpn_protocols, uint16_t num_alpn_protocols,
-    tsi_ssl_client_handshaker_factory** factory);
+    const char* pem_root_certs, const ssl_root_certs_store* root_store,
```

Comment:
I would actually prefer that we don't change this *public* API. What would be nice is to have `tsi_create_ssl_client_handhaker_factory_with_options` using an option struct with the existing params (+ the store) and deprecate the old one.